Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/HostingConfiguration.cs

```diff
@@ -63,7 +63,7 @@ public HostingConfiguration ([NotNull] IWebTestSettings webTestSettings, [NotNul
 
     public IHostingStrategy GetHostingStrategy ()
     {
-      if (string.IsNullOrEmpty(_hostingSettings.Type))
+      if (string.IsNullOrEmpty(_hostingSettings.Type) || _hostingSettings.Type is "None" or "Null")
```

Comment:
1. Do we want to support both strings? usually, I only use `None`.
2. Please add this also to the XML schema for the xml config.
3. Do we also need to do something for the json config?